src/main/resources/application.properties

```diff
@@ -1 +1,25 @@
 
+
+# datasource PostgreSQl
+spring.sql.init.platform=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/techiteasy2
+spring.datasource.username=postgres
+spring.datasource.password=password
+spring.datasource.driver-class-name=org.postgresql.Driver
+server.port=8081
+
+# jpa
+spring.jpa.database=postgresql
+spring.jpa.show-sql=true
+
+# generate schema ddl to create tables
+spring.jpa.generate-ddl=true
+spring.jpa.hibernate.ddl-auto=create
+
+
+# database initialization with data.sql after hibernate
+spring.sql.init.mode=always
+spring.jpa.defer-datasource-initialization=true
+
+# hibernate
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
```

Comment:
Mooi de structuur uit de les van Robert-Jan overgenomen. Ziet er goed uit. 

Wellicht heb je het al gezien, maar in de uitwerkingen staat helemaal uitgeschreven waar elke regel voor is. Misschien interessant om nog eens te bekijken: https://github.com/hogeschoolnovi/backend-spring-boot-tech-it-easy-model-uitwerkingen/blob/springboot3.0/src/main/resources/application.properties